stretch-oracle-java8/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM debian:stretch
```

Comment:
is there any reason why we couldn't use the `slim` flavour of this image?

Reply:
because it is called `stretch-oracle`. In the next step I was gonna add `stretch-slim-oracle` as well as openjdk flavours.